From 2364c7140f9a34e969d07dbb3a26ec1dac168cfb Mon Sep 17 00:00:00 2001
From: Hu Kejun <william.hu@rock-chips.com>
Date: Sun, 13 Dec 2020 11:14:50 +0800
Subject: [PATCH] media: rockchip: ispp: fix compile error in
 rkispp_compat_ioctl32

fix compile error: implicit declaration of function 'rkisp_ioctl'

Signed-off-by: Hu Kejun <william.hu@rock-chips.com>
Change-Id: Ic4822eb6697d300c76d0845d90284ae5940caa4a
---
 drivers/media/platform/rockchip/ispp/ispp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/rockchip/ispp/ispp.c b/drivers/media/platform/rockchip/ispp/ispp.c
index a154f2a2b98a..113731f2920a 100644
--- a/drivers/media/platform/rockchip/ispp/ispp.c
+++ b/drivers/media/platform/rockchip/ispp/ispp.c
@@ -416,7 +416,7 @@ static long rkispp_compat_ioctl32(struct v4l2_subdev *sd,
 	case RKISPP_CMD_SET_FECBUF_SIZE:
 		ret = copy_from_user(&fecsize, up, sizeof(fecsize));
 		if (!ret)
-			ret = rkisp_ioctl(sd, cmd, &fecsize);
+			ret = rkispp_ioctl(sd, cmd, &fecsize);
 		break;
 	default:
 		ret = -ENOIOCTLCMD;
-- 
2.35.3

